frontend/src/components/getStarted/getStarted.jsx

```diff
@@ -7,13 +7,17 @@ import Demo from './Demo.jsx';
 import Marketplace from './Marketplace.jsx';
 import Blog from './Blog.jsx';
 import GitHub from './GitHub.jsx';
-
+import StagingBanner from './StagingBanner.jsx'
 
 export default class GetStartedComponent extends React.Component {
-
+    constructor(props){
+      super(props);
+      this.host = window.location.hostname;
+    }
     render () {
       return (
         <div>
+          {this.host == "staging.ninja" && <StagingBanner />}

```

Comment:
This is a sexy as hell one-liner

Reply:
A compliment on my front end code? *swoons*

Reply:
hell to da yeh!